proxysql/datadog_checks/proxysql/proxysql.py

```diff
@@ -58,8 +58,12 @@ def __init__(self, name, init_config, instances):
         self.tags.append("proxysql_server:{}".format(self.host))
         self.tags.append("proxysql_port:{}".format(self.port))
 
+        self.version_metadata = is_affirmative(self.instance.get('version_metadata', False))
```

Comment:
I guess that you're not implementing a default at `true` with current code.
So, I suggest:
```suggestion
        self.version_metadata = is_affirmative(self.instance.get('version_metadata', True))
```